### PR TITLE
Update m365-campaigns-increase-protection.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation for enabling unified Audit log

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3704

File: microsoft-365/campaigns/m365-campaigns-increase-protection.md [hunk 1]
```diff
@@ -253,8 +253,8 @@ After you turn on the audit log search in the Security & Compliance Center, you
 
 You must be assigned the Audit Logs role in Exchange Online to turn audit log search on or off in your Microsoft 365 subscription. By default, this role is assigned to the Compliance Management and Organization Management role groups on the Permissions page in the Exchange admin center. Global admins in Microsoft 365 are members of this group by default.
 
-1. To turn on the audit log search on, go to the admin center at <https://admin.microsoft.com> and then choose **Compliance** under **Admin centers** in the left nav.
-2. On the **Microsoft 365 compliance** page, choose **More resources**, and then **Open** on the **Office 365 security & center** card.
+1. To turn on the audit log search on, go to the admin center at <https://admin.microsoft.com> and then choose **Security** under **Admin centers** in the left nav.
```
Comment (JohanFreelancer9) [inline microsoft-365/campaigns/m365-campaigns-increase-protection.md]:
```suggestion
1. To turn on the audit log search, go to the admin center at <https://admin.microsoft.com> and then choose **Security** under **Admin centers** in the left nav.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Pull request merged**